### PR TITLE
Merge to live 1 June 2020

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@wadepickett  can you approve so I can S&M?

File: iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md [hunk 1]
```diff
@@ -35,44 +37,44 @@ In order for PHP to run properly on IIS there is a set of recommended settings t
 
 When some configuration settings are configured incorrectly you will see a warning message when you open PHP Manager:
 
-![](http://download.codeplex.com/Project/Download/FileDownload.aspx?ProjectName=phpmanager&amp;DownloadId=153524)
+![](http://www.phpmanager.xyz/latest/_images/warning.png)
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md:40]: nit: alt-text on images. Well, at least on new/recently modified content

File: iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md [hunk 2]
```diff
@@ -35,44 +37,44 @@ In order for PHP to run properly on IIS there is a set of recommended settings t
...
 
 After clicking on "View Recommendations" link you can see all the configuration issues that exist in your PHP installation. You can review each issue description and the recommended corrective action. You can also select which configuration issues you want to be fixed:
 
-![](http://download.codeplex.com/Project/Download/FileDownload.aspx?ProjectName=phpmanager&amp;DownloadId=153525)
+![](http://www.phpmanager.xyz/latest/_images/fixed.png)
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md:44]: nit: alt-text

File: iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md [hunk 3]
```diff
@@ -35,44 +37,44 @@ In order for PHP to run properly on IIS there is a set of recommended settings t
...
 
 ### Switching between PHP versions
 
 After multiple PHP versions have been registered with IIS, you can use PHP Manager to easily switch between the versions on a server and site level. This means that you can configure some IIS sites to use one PHP version, while other sites use different version.
 
-![](http://download.codeplex.com/Project/Download/FileDownload.aspx?ProjectName=phpmanager&amp;DownloadId=145958)
+![](http://www.phpmanager.xyz/latest/_images/switch.png)
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md:50]: nit: alt-text

File: iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md [hunk 4]
```diff
@@ -35,44 +37,44 @@ In order for PHP to run properly on IIS there is a set of recommended settings t
...
 
 ### Checking phpinfo() output
 
 phpinfo() function in PHP provides very detailed information about all aspects of PHP runtime configuration. To check the phpinfo() output from within PHP Manager use the "**Check phpinfo()**" task.
 
-![](http://download.codeplex.com/Project/Download/FileDownload.aspx?ProjectName=phpmanager&amp;DownloadId=145959)
+![](http://www.phpmanager.xyz/latest/_images/phpinfo.png)
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md:56]: nit: alt-text

File: iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md [hunk 5]
```diff
@@ -35,44 +37,44 @@ In order for PHP to run properly on IIS there is a set of recommended settings t
...
 
 ### Configuring Error Reporting
 
 You can use PHP Manager to configure error reporting level in PHP. If IIS is used on a development machine then you may want to use verbose error reporting in order to see all the errors, warnings and notices from your PHP application right away. If IIS is used as a production server, then error reporting level is less verbose and errors are logged in a log file, but never communicated to HTTP client.
 
-![](http://download.codeplex.com/Project/Download/FileDownload.aspx?ProjectName=phpmanager&amp;DownloadId=145984)
+![](http://www.phpmanager.xyz/latest/_images/error.png)
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md:62]: nit: alt-text

File: iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md [hunk 6]
```diff
@@ -35,44 +37,44 @@ In order for PHP to run properly on IIS there is a set of recommended settings t
...
 
 ### Configuring Runtime Limits
 
 Various timeouts and limits can be configured in the "**PHP Runtime Limits**" page:
 
-![](http://download.codeplex.com/Project/Download/FileDownload.aspx?ProjectName=phpmanager&amp;DownloadId=146232)
+![](http://www.phpmanager.xyz/latest/_images/limits.png)
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md:68]: nit: alt-text

File: iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md [hunk 7]
```diff
@@ -35,44 +37,44 @@ In order for PHP to run properly on IIS there is a set of recommended settings t
...
 
 ### Configuring All PHP Settings
 
 All existing PHP settings can be added, removed and modified in the "**PHP Settings**" page.
 
-![](http://download.codeplex.com/Project/Download/FileDownload.aspx?ProjectName=phpmanager&amp;DownloadId=146015)
+![](http://www.phpmanager.xyz/latest/_images/settings.png)
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md:74]: nit: alt-text

File: iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md [hunk 8]
```diff
@@ -35,44 +37,44 @@ In order for PHP to run properly on IIS there is a set of recommended settings t
...
 
 ### Enabling or Disabling PHP Extensions
 
 PHP extensions can be enabled or disabled in the "**PHP Extensions**" page:
 
-![](http://download.codeplex.com/Project/Download/FileDownload.aspx?ProjectName=phpmanager&amp;DownloadId=145987)
+![](http://www.phpmanager.xyz/latest/_images/extentions.png)
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-on-iis/using-php-manager-for-iis-to-setup-and-configure-php.md:80]: nit: alt-text

**Comment by Rick-Anderson** [comment]
@shirhatti  I'm not approved to merge to live. Can we do the missing `alt-text` in another PR?

**Comment by wadepickett** [comment]
@Rick-Anderson, I added approval.

**Pull request merged**